setup.py

```diff
@@ -125,7 +129,7 @@ class ONNXCommand(setuptools.Command):
     user_options = []
 
     def initialize_options(self):
-        pass
+        self.onnxml=ONNX_ML
```

Comment:
You are using `ONNX_ML` as a global variable, so no need to store it into separate command.

Reply:
fixed. Thanks!